schematic/store/synapse.py

```diff
@@ -291,8 +291,19 @@ def query_fileview(
                 self.storageFileviewTable = self.syn.tableQuery(
                     query=self.fileview_query,
                 ).asDataFrame()
-            except SynapseHTTPError:
-                raise AccessCredentialsError(self.storageFileview)
+            except SynapseHTTPError as exc:
+                exception_text = str(exc)
+                if "Unknown column path" in exception_text:
+                    raise ValueError(
+                        "The path column has not been added to the fileview. Please make sure that the fileview is up to date. You can add the path column to the fileview by follwing the instructions in the validation rules documentation (https://sagebionetworks.jira.com/wiki/spaces/SCHEM/pages/2645262364/Data+Model+Validation+Rules#Filename-Validation)."
```

Comment:
hmm should the link of documentation be included in our code? I feel like this might be hard to maintain in the future? If the documentation moves we might forget to update the code. Also, if we are not expecting users to see it, then we probably shouldn't include it? External users can't click on to Sage's confluence page anyway

Reply:
Yeah that's a good point; I'll remove